### PR TITLE
Input & snackbar

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
Please provide review

### CODE REVIEW

File: component/input/input.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.input-box{
+    position: relative;
+    background-color: transparent;
+    border-radius: 5px;
+    padding: 6px 6px 6px 6px;
```
Comment (dhimanaishwarya) [inline component/input/input.css]: you can just give padding: 6px; here

File: component/input/input.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                <input type="password" class="input-box" placeholder="Enter password">
+                <input type="date" class="input-box" placeholder="Enter date of birth">
+                <input type="file" class="input-box" placeholder="upload file">
+            </div>
+            
+            <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=base16-dark&wt=none&l=htmlmixed&width=955&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cinput%2520type%253D%2522text%2522%2520class%253D%2522input-box%2522%2520placeholder%253D%2522Enter%2520email%2520address%2522%253E%250A%253Cinput%2520type%253D%2522password%2522%2520class%253D%2522input-box%2522%2520placeholder%253D%2522Enter%2520password%2522%253E%250A%253Cinput%2520type%253D%2522date%2522%2520class%253D%2522input-box%2522%2520placeholder%253D%2522Enter%2520date%2520of%2520birth%2522%253E%250A%253Cinput%2520type%253D%2522file%2522%2520class%253D%2522input-box%2522%2520placeholder%253D%2522upload%2520file%2522%253E"
+  style="width: 955px; height: 148px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (dhimanaishwarya) [inline component/input/input.html]: would suggest to add a class for styling iframe to maintain uniformity .
Rest LGTM

**Review by j24m — APPROVED** [review]
Honestly, you can make changes suggested by others , but to me your code looks good.
Good work:)

**Review by m26unkwn — APPROVED** [review]

File: component/input/input.css (hunk 1 above)
Comment (dev-ashishsethi) [reply component/input/input.css]: oh I missed it! Thanks for letting me know

File: component/input/input.html (hunk 2 above)
Comment (dev-ashishsethi) [reply component/input/input.html]: done! Thanks

File: component.css [hunk 3]
```diff
@@ -93,20 +102,24 @@ h1,h2,h3,h4,h5,h6{
...
-
 .list-items{
     padding: 1rem;
     border-radius: 10px;
     
 }
 .list-items:hover{
-    background-color: grey;
+    padding-left: 2rem;
 }
```
Comment (manojsarna) [inline component.css]: color varibales can be used.!! if you are using it elsewhere.!!

File: component.css [hunk 4]
```diff
@@ -130,6 +143,15 @@ a:visited{
...
+    padding: 0rem;
+}
+.social-links{
+    float: left;
+    padding: 5px;
+}
+
 .text{
     background-color: var(--white);
     margin: 0rem 1rem;
```
Comment (manojsarna) [inline component.css:183]: pixel can be used.!!

**Review by dhruvin25799 — APPROVED** [review]
Looks good to me.
Maybe instead of the snack bar appearing on the bottom right try making it centered horizontally.

File: component/snackbar/snackbar.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.liveDemo-btn{
+    display: block;
+    background-color: var(--red);
+    border: none;
+    margin-top: 1rem;
+    color: var(--white);
+    padding: 8px;
+    border-radius: 5px;
```
Comment (manojsarna) [inline component/snackbar/snackbar.css:21]: use correct naming conventions!!

**Review by manojsarna — APPROVED** [review]
overalll code looks good!!

good to merge.!!

File: component.css [hunk 6]
```diff
@@ -1,9 +1,15 @@
 @import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Nunito&family=Roboto:wght@100;300;400;500;700&display=swap');
 @import url('components\\card\\card.css') ;
+@import url('component\\input\\input.css') ;
+@import url('component\\snackbar\\snackbar.css') ;
+
 :root{
     --black:rgb(20 20 20);
```
Comment (deekshithmd) [inline component.css]: I think it is better to use color code

File: component/snackbar/snackbar.html [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+                                d="M22.46 6c-.77.35-1.6.58-2.46.69c.88-.53 1.56-1.37 1.88-2.38c-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29c0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15c0 1.49.75 2.81 1.91 3.56c-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07a4.28 4.28 0 0 0 4 2.98a8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21C16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56c.84-.6 1.56-1.36 2.14-2.23z"
+                                fill="currentColor"></path>
+                        </svg></a>
+                </li>
+            </ul>
+        </div>
+
+        <div class="main-content">
+
+             <h1>Snackbar</h1>
```
Comment (deekshithmd) [inline component/snackbar/snackbar.html:113]: check the alignment of tags, otherwise code looks good

**Pull request merged**